### PR TITLE
Record test results using Appveyor's testresults API.

### PR DESCRIPTION
**Pull request by rryan**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -53,6 +53,7 @@ on_success:
 on_failure:
   - echo "*** FAILURE ***"
 on_finish:
+  - ps: (new-object net.webclient).UploadFile("https://ci.appveyor.com/api/testresults/junit/$($env:APPVEYOR_JOB_ID)", (Resolve-Path .\test_results.xml))
```
Comment (sblaisot) [inline appveyor.yml]: Is it `(new-object net.webclient)`or `(new-object 'System.Net.WebClient') ?

Comment (rryan) [inline appveyor.yml]: Hm, not sure -- I copied this from:
https://github.com/mgerhardy/caveexpress/blob/master/appveyor.yml#L53

Looks like net.webclient may work (assuming case insensitive):
https://www.appveyor.com/docs/how-to/download-file/#webclient-class

Comment (sblaisot) [inline appveyor.yml]: Yes, I confirm case incensitive.

Appveyor example uses system.net.webclient
https://www.appveyor.com/docs/running-tests/#uploading-xml-test-results

nervermind, if it works...

**Comment by sblaisot** [comment]
LGTM

**Comment by sblaisot** [comment]
Can you also check the "Rolling builds" checkbox in the project settings->general page ?
This will cancel all scheduled build in the same branch or PR when a new commit arrive to start building the latest commit earlier.

**Comment by rryan** [comment]
> Can you also check the "Rolling builds" checkbox in the project settings->general page ?
> This will cancel all scheduled build in the same branch or PR when a new commit arrive to start building the latest commit earlier.

Done

**Comment by rryan** [comment]
Cool :)
https://ci.appveyor.com/project/mixxxdj/mixxx/build/master-14/job/5f0f8htxb9gmhc9o/tests

**Pull request merged**